js-api/src/chem.ts

```diff
@@ -175,6 +175,9 @@ export namespace chem {
       return this._sketcherTypeChanged;
     }
 
+    get calculating(): boolean {return this.loader.style.display == 'initial';}
```

Comment:
Are you setting it to "initial" somewhere? Because I think otherwise it's going to be null

Reply:
Hm... I used exactly the same approach as our substructure filter does. In filter we do not set initial anywhere and everything works as it should. So I guess initial is set somewhere in loader constructor.

Reply:
Well, double-check it please. As you can see we already had an issue with the loader, this can be related.

Reply:
In loader constructor we set class 'grok-loader' on a div element. And initial state for display there is 'inline-block'.

Reply:
I still have a problem with it and still don't get how it works. The unassigned style is "", and not "initial". If anything, you should check for the opposite (custom style).